R/anti_join-rd.R

```diff
@@ -0,0 +1,14 @@
+#' @title Filtering joins (duckplyr)
+#'
+#' @description  This is a method for the [`dplyr::anti_join()`] generic.
+#' `anti_join()` return all rows from `x` with**out** a match in `y`.
+#'
+#' @inheritParams dplyr::anti_join
+#' @examples
+#' library("dplyr") # for the data
+#' library("duckplyr")
+#' band_members %>% anti_join(band_instruments)
```

Comment:
Can we use a duckplyr example here? We can do example-driven development -- if an example feels awkward to implement, the code is probably awkward to use.

Reply:
what do you mean by duckplyr example?

Reply:
I might be mistaken. With the current implementation, this already uses duckplyr because it is attached, so perhaps no need to act here. We could also make it explicit by calling `duckplyr::as_duckplyr_tibble()` on the data.

No need to act now.